packages/core/src/components/text/_text.scss

```diff
@@ -6,8 +6,7 @@
 /*
 Text
 
-A component used to represent textual content.
-TODO BP add the docs you want here, I don't have the vision for this
+A component used to represent textual content with optional overflow behavior.
```

Comment:
I feel like we're repeating ourselves here and in the prop description of ellipsize. but I made the change